corehq/ex-submodules/pillowtop/tests/test_elasticsearch.py

```diff
@@ -195,6 +205,17 @@ def test_replace_doc(self):
         del doc['property']
         self._send_to_es_and_check(doc, update=True)
 
+    def test_merge_doc(self):
+        doc = {'_id': uuid.uuid4().hex, 'doc_type': 'MyCoolDoc', 'property': 'bar'}
+        self._send_to_es_and_check(doc)
+
+        update = doc.copy()
+        del update['property']
+        update['new_prop'] = 'new_val'
+        self._send_to_es_and_check(update, update=True)
+        # merging should still keep old 'property'
+        self._send_to_es_and_check(update, update=True, es_merge_update=True)
```

Comment:
Don't think this test is actually working. `self._send_to_es_and_check(update, update=True)` will replace the doc with the new one so `self._send_to_es_and_check(update, update=True, es_merge_update=True)` doesn't actually test the merging.

I think you just want to do something like this
```python
self._send_to_es_and_check('_id': doc['_id'], 'new_prop': 'new_val', update=True, es_merge_update=True)
# Doc in ES should contain new prop and old prop - make sure old prop is still there.
updated_doc = self.es.get_source(self.index, TEST_INDEX_INFO.type, doc['_id'])
self.assertIn('property', doc)
self.assertEqual(doc['property'], 'bar')
```